src/libraries/System.Net.Security/src/System/Net/Security/SslStreamCertificateContext.Linux.cs

```diff
@@ -115,13 +116,15 @@ private SslStreamCertificateContext(X509Certificate2 target, ReadOnlyCollection<
 
                 if (task.IsCompletedSuccessfully)
                 {
+                    if (NetEventSource.Log.IsEnabled()) NetEventSource.Info(this, $"Got OCSP response.");
                     return task.Result;
                 }
             }
             catch
             {
             }
 
+            if (NetEventSource.Log.IsEnabled()) NetEventSource.Info(this, "No OCSP response available.");
```

Comment:
All of these logging statements violate style guidelines.  Guidelines 1 and 18.1:

>18. When using a single-statement if, we follow these conventions:
>  *  Never use single-line form (for example: if (source == null) throw new ArgumentNullException("source");)

Reply:
Should be good now